tools/ports/bullet.py

```diff
@@ -32,10 +32,7 @@ def create(final):
         shared.safe_ensure_dirs(os.path.dirname(target))
         shutil.copyfile(fullpath, target)
 
-    includes = []
-    for base, dirs, _ in os.walk(src_path, topdown=False):
-      for dir in dirs:
-        includes.append(os.path.join(base, dir))
+    includes = [os.path.join(base, dir) for base, dirs, _ in os.walk(src_path, topdown=False) for dir in dirs]
```

Comment:
Can you revert this one?  I think it makes the line to long.

Reply:
```suggestion
    for base, dirs, _ in os.walk(src_path, topdown=False):
      for dir in dirs:
        includes.append(os.path.join(base, dir))
```